# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../digikam/mysql.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur digikam-1.7.0-orig/CMakeLists.txt digikam-1.7.0/CMakeLists.txt
--- digikam-1.7.0-orig/CMakeLists.txt	2010-12-19 23:59:40.804061885 +0100
+++ digikam-1.7.0/CMakeLists.txt	2010-12-20 00:00:39.288063218 +0100
@@ -200,7 +200,7 @@
 #       under MACOS-X, through macports project, mysql_install_db PERL script is named "mysql_install_db5"
 FIND_PROGRAM( INITCMD_MYSQL mysql_install_db mysql_install_db5 mysql_install_db.pl
               PATHS /usr/sbin /usr/local/sbin /usr/libexec /usr/local/libexec
-                    /opt/mysql/libexec /usr/mysql/bin /opt/mysql/bin ${MYSQL_TOOLS_PATH}
+                    /opt/mysql/libexec /opt/mysql/scripts /usr/mysql/bin /opt/mysql/bin ${MYSQL_TOOLS_PATH}
                     ENV MYSQL_TOOLS_PATH )
 MESSAGE(STATUS "Found MySQL install_db executable at: ${INITCMD_MYSQL}")
 
